scripts/generate_workers_cfg.sh

```diff
@@ -101,20 +117,50 @@ generate_workers () {
             if [ $(_jq '.gpu') = "true" ]; then
                 add_string_env_var NVIDIA_VISIBLE_DEVICES all
             fi
-            add_section networks
-            add_item mediacloudai_global
-            add_item workers
+
+            if [ $2 = "master" ]; then
```

Comment:
Did you rename 'master' config by 'startup'

Reply:
Because in the README file, I think you are mentionning only dev and startup config name

Reply:
True ! Gonna correct it !